sources/VVPSDK.swift

```diff
@@ -406,12 +406,13 @@ public struct VVPSDK {
         }
         
         func setupVRMWithNewCore() {
+            let maxRedirectCount = 2
```

Comment:
WTF?

Reply:
Have to be put this on M/S side - make necessary arrangements on client side.

Reply:
It's separate task, will do this in other PR.